include/config/pokemon.h

```diff
@@ -53,6 +53,8 @@
 #define P_ONLY_OBTAINABLE_SHINIES        FALSE       // If TRUE, Pokémon encountered in the Battle Pyramid won't be shiny.
 #define P_NO_SHINIES_WITHOUT_POKEBALLS   FALSE       // If TRUE, Pokémon encountered when the player is out of Poké Balls won't be shiny
 #define P_SHOW_DYNAMIC_TYPES             FALSE       // If TRUE, all moves with dynamic type changes will be reflected as their current type in battle/summary screens instead of just select ones like in vanilla.
+#define P_SUMMARY_SCREEN_MOVE_RELEARNER  FALSE       // If TRUE, shows an option for Pokémon to relearn moves on the summary screen moves page.
+#define P_SUMMARY_SCREEN_RENAME          FALSE       // If TRUE, an option to change Pokémon nicknames replaces the cancel prompt on the summary screen info page.
```

Comment:
since both of these are a SV feature would it not make sense to have it on by default?

Reply:
Flipped these to true.